source/index.ts

```diff
@@ -318,4 +332,15 @@ export default class PQueue<QueueType extends Queue<EnqueueOptionsType> = Priori
 	get isPaused(): boolean {
 		return this._paused;
 	}
+
+	/**
+	Set the timeout for future operations.
+	*/
+	set timeout(milliseconds: number | undefined) {
```

Comment:
```suggestion
	set timeout(milliseconds?: number) {
```

Reply:
I had to revert this btw because accessors demand the same type but it did not treat `?: number` and `number | undefined` equally here. Too bad we can't just use Swift's Optional-style for return types but just for parameters and fields

Reply:
@ltetzlaff see https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters

Reply:
what for? I'm aware of the concept of optional parameters, I'm just saying it is not applicable for accessors

Reply:
Ah, missed that! Thanks for pointing out :P